src/core/utils/RbSettings.h

```diff
@@ -30,6 +30,7 @@ class RbSettings {
     double                      getTolerance(void) const;                           //!< Retrieve the tolerance for comparing doubles
     bool                        getUseScaling(void) const;                          //!< Retrieve the flag whether we should scale the likelihood in CTMC models
     int                         getDebugMCMC(void) const;                           //!< How much work should we perform to check MCMC?
+    int                         getLogMCMC(void) const;                           //!< How much work should we perform to check MCMC?
```

Comment:
Maybe change the comment to clarify the distinction b/w the two

Reply:
Fixed